Makefile

```diff
@@ -83,6 +84,9 @@ SRCS := $(shell find $(SRC_DIRS) -name *.cpp -or -name *.c -or -name *.s)
 
 OBJS := $(subst $(TOP_DIR),$(BUILD_DIR),$(SRCS:.c=.o))
 
+# Remove duplicates using sort
+OBJS := $(sort $(OBJS))
```

Comment:
why would there be duplicates? Something is wrong if we need to-do this.

Sort will not remove duplicates, it will group them together.

Reply:
I get multiple definition error w/o sort, while binary is created.
```
Building [framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c]
Linking test /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/ut-test
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/CUError.o: in function `CU_set_error':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/CUError.c:64: multiple definition of `CU_set_error'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/CUError.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/CUError.c:64: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/CUError.o: in function `CU_get_error':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/CUError.c:81: multiple definition of `CU_get_error'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/CUError.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/CUError.c:81: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/CUError.o: in function `CU_get_error_msg':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/CUError.c:87: multiple definition of `CU_get_error_msg'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/CUError.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/CUError.c:87: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/CUError.o: in function `CU_set_error_action':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/CUError.c:93: multiple definition of `CU_set_error_action'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/CUError.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/CUError.c:93: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/CUError.o: in function `CU_get_error_action':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/CUError.c:99: multiple definition of `CU_get_error_action'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/CUError.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/CUError.c:99: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_assertImplementation':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:157: multiple definition of `CU_assertImplementation'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:157: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_set_suite_start_handler':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:181: multiple definition of `CU_set_suite_start_handler'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:181: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_set_test_start_handler':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:187: multiple definition of `CU_set_test_start_handler'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:187: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_set_test_complete_handler':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:193: multiple definition of `CU_set_test_complete_handler'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:193: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_set_suite_complete_handler':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:199: multiple definition of `CU_set_suite_complete_handler'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:199: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_set_all_test_complete_handler':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:205: multiple definition of `CU_set_all_test_complete_handler'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:205: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_set_suite_init_failure_handler':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:211: multiple definition of `CU_set_suite_init_failure_handler'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:211: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_set_suite_cleanup_failure_handler':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:217: multiple definition of `CU_set_suite_cleanup_failure_handler'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:217: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_get_suite_start_handler':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:223: multiple definition of `CU_get_suite_start_handler'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:223: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_get_test_start_handler':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:229: multiple definition of `CU_get_test_start_handler'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:229: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_get_test_complete_handler':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:235: multiple definition of `CU_get_test_complete_handler'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:235: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_get_suite_complete_handler':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:241: multiple definition of `CU_get_suite_complete_handler'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:241: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_get_all_test_complete_handler':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:247: multiple definition of `CU_get_all_test_complete_handler'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:247: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_get_suite_init_failure_handler':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:253: multiple definition of `CU_get_suite_init_failure_handler'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:253: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_get_suite_cleanup_failure_handler':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:259: multiple definition of `CU_get_suite_cleanup_failure_handler'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:259: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_get_number_of_suites_run':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:265: multiple definition of `CU_get_number_of_suites_run'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:265: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_get_number_of_suites_failed':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:271: multiple definition of `CU_get_number_of_suites_failed'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:271: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_get_number_of_suites_inactive':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:277: multiple definition of `CU_get_number_of_suites_inactive'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:277: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_get_number_of_tests_run':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:283: multiple definition of `CU_get_number_of_tests_run'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:283: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_get_number_of_tests_failed':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:289: multiple definition of `CU_get_number_of_tests_failed'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:289: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_get_number_of_tests_inactive':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:295: multiple definition of `CU_get_number_of_tests_inactive'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:295: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_get_number_of_asserts':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:301: multiple definition of `CU_get_number_of_asserts'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:301: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_get_number_of_successes':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:307: multiple definition of `CU_get_number_of_successes'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:307: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_get_number_of_failures':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:313: multiple definition of `CU_get_number_of_failures'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:313: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_get_number_of_failure_records':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:319: multiple definition of `CU_get_number_of_failure_records'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:319: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_get_elapsed_time':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:325: multiple definition of `CU_get_elapsed_time'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:325: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_get_failure_list':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:336: multiple definition of `CU_get_failure_list'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:336: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_get_run_summary':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:342: multiple definition of `CU_get_run_summary'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:342: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_run_all_tests':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:348: multiple definition of `CU_run_all_tests'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:348: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_run_suite':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:387: multiple definition of `CU_run_suite'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:387: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_run_test':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:419: multiple definition of `CU_run_test'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:419: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_clear_previous_results':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:512: multiple definition of `CU_clear_previous_results'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:512: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_get_current_suite':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:518: multiple definition of `CU_get_current_suite'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:518: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_get_current_test':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:524: multiple definition of `CU_get_current_test'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:524: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_is_test_running':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:530: multiple definition of `CU_is_test_running'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:530: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_set_fail_on_inactive':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:536: multiple definition of `CU_set_fail_on_inactive'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:536: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_get_fail_on_inactive':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:542: multiple definition of `CU_get_fail_on_inactive'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:542: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_print_run_results':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:548: multiple definition of `CU_print_run_results'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:548: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o: in function `CU_get_run_results_string':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:565: multiple definition of `CU_get_run_results_string'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestRun.c:565: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/Util.o: in function `CU_translate_special_characters':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/Util.c:96: multiple definition of `CU_translate_special_characters'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/Util.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/Util.c:96: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/Util.o: in function `CU_translated_strlen':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/Util.c:167: multiple definition of `CU_translated_strlen'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/Util.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/Util.c:167: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/Util.o: in function `CU_compare_strings':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/Util.c:186: multiple definition of `CU_compare_strings'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/Util.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/Util.c:186: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/Util.o: in function `CU_trim':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/Util.c:200: multiple definition of `CU_trim'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/Util.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/Util.c:200: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/Util.o: in function `CU_trim_left':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/Util.c:207: multiple definition of `CU_trim_left'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/Util.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/Util.c:207: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/Util.o: in function `CU_trim_right':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/Util.c:228: multiple definition of `CU_trim_right'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/Util.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/Util.c:228: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/Util.o: in function `CU_number_width':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/Util.c:246: multiple definition of `CU_number_width'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/Util.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/Util.c:246: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_initialize_registry':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:95: multiple definition of `CU_initialize_registry'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:95: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_cleanup_registry':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:122: multiple definition of `CU_cleanup_registry'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:122: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_create_new_registry':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:933: multiple definition of `CU_create_new_registry'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:933: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_registry_initialized':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:116: multiple definition of `CU_registry_initialized'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:116: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_destroy_existing_registry':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:946: multiple definition of `CU_destroy_existing_registry'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:946: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_get_registry':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:133: multiple definition of `CU_get_registry'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:133: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_set_registry':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:139: multiple definition of `CU_set_registry'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:139: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_add_suite_with_setup_and_teardown':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:151: multiple definition of `CU_add_suite_with_setup_and_teardown'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:151: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_add_suite':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:182: multiple definition of `CU_add_suite'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:182: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_set_suite_active':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:188: multiple definition of `CU_set_suite_active'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:188: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_set_suite_name':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:204: multiple definition of `CU_set_suite_name'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:204: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_set_suite_initfunc':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:225: multiple definition of `CU_set_suite_initfunc'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:225: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_set_suite_cleanupfunc':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:241: multiple definition of `CU_set_suite_cleanupfunc'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:241: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_get_suite':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:257: multiple definition of `CU_get_suite'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:257: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_get_suite_by_name':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:960: multiple definition of `CU_get_suite_by_name'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:960: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_get_suite_at_pos':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:277: multiple definition of `CU_get_suite_at_pos'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:277: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_get_suite_by_index':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:981: multiple definition of `CU_get_suite_by_index'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:981: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_get_suite_pos':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:294: multiple definition of `CU_get_suite_pos'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:294: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_get_suite_pos_by_name':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:323: multiple definition of `CU_get_suite_pos_by_name'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:323: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_add_test':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:352: multiple definition of `CU_add_test'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:352: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_set_test_active':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:390: multiple definition of `CU_set_test_active'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:390: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_set_test_name':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:406: multiple definition of `CU_set_test_name'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:406: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_set_test_func':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:427: multiple definition of `CU_set_test_func'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:427: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_get_test':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:443: multiple definition of `CU_get_test'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:443: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_get_test_by_name':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:999: multiple definition of `CU_get_test_by_name'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:999: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_get_test_at_pos':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:466: multiple definition of `CU_get_test_at_pos'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:466: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_get_test_by_index':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:1020: multiple definition of `CU_get_test_by_index'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:1020: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_get_test_pos':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:486: multiple definition of `CU_get_test_pos'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:486: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_get_test_pos_by_name':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:518: multiple definition of `CU_get_test_pos_by_name'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:518: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_register_nsuites':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:572: multiple definition of `CU_register_nsuites'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:572: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o: in function `CU_register_suites':
/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:604: multiple definition of `CU_register_suites'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.o:/home/jpn323/workspace/gh96-ut-core/framework/CUnit-2.1-3/CUnit/Sources/Framework/TestDB.c:604: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/ut_main.o: in function `UT_init':
/home/jpn323/workspace/gh96-ut-core/src/ut_main.c:170: multiple definition of `UT_init'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/ut_main.o:/home/jpn323/workspace/gh96-ut-core/src/ut_main.c:170: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/ut_main.o: in function `UT_exit':
/home/jpn323/workspace/gh96-ut-core/src/ut_main.c:186: multiple definition of `UT_exit'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/ut_main.o:/home/jpn323/workspace/gh96-ut-core/src/ut_main.c:186: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_GetErrorPtr':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:95: multiple definition of `cJSON_GetErrorPtr'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:95: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_GetStringValue':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:100: multiple definition of `cJSON_GetStringValue'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:100: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_IsString':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2947: multiple definition of `cJSON_IsString'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2947: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_GetNumberValue':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:110: multiple definition of `cJSON_GetNumberValue'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:110: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_IsNumber':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2937: multiple definition of `cJSON_IsNumber'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2937: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_Version':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:125: multiple definition of `cJSON_Version'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:125: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_InitHooks':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:210: multiple definition of `cJSON_InitHooks'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:210: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_Delete':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:254: multiple definition of `cJSON_Delete'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:254: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_SetNumberHelper':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:383: multiple definition of `cJSON_SetNumberHelper'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:383: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_SetValuestring':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:401: multiple definition of `cJSON_SetValuestring'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:401: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_free':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:3117: multiple definition of `cJSON_free'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:3117: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_ParseWithOpts':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:1080: multiple definition of `cJSON_ParseWithOpts'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:1080: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_ParseWithLengthOpts':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:1096: multiple definition of `cJSON_ParseWithLengthOpts'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:1096: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_Parse':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:1176: multiple definition of `cJSON_Parse'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:1176: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_ParseWithLength':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:1181: multiple definition of `cJSON_ParseWithLength'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:1181: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_Print':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:1253: multiple definition of `cJSON_Print'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:1253: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_PrintUnformatted':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:1258: multiple definition of `cJSON_PrintUnformatted'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:1258: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_PrintBuffered':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:1263: multiple definition of `cJSON_PrintBuffered'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:1263: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_PrintPreallocated':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:1293: multiple definition of `cJSON_PrintPreallocated'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:1293: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_GetArraySize':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:1829: multiple definition of `cJSON_GetArraySize'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:1829: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_GetArrayItem':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:1871: multiple definition of `cJSON_GetArrayItem'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:1871: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_GetObjectItem':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:1913: multiple definition of `cJSON_GetObjectItem'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:1913: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_GetObjectItemCaseSensitive':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:1918: multiple definition of `cJSON_GetObjectItemCaseSensitive'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:1918: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_HasObjectItem':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:1923: multiple definition of `cJSON_HasObjectItem'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:1923: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_AddItemToArray':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:1991: multiple definition of `cJSON_AddItemToArray'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:1991: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_AddItemToObject':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2049: multiple definition of `cJSON_AddItemToObject'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2049: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_AddItemToObjectCS':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2055: multiple definition of `cJSON_AddItemToObjectCS'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2055: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_AddItemReferenceToArray':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2060: multiple definition of `cJSON_AddItemReferenceToArray'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2060: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_AddItemReferenceToObject':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2070: multiple definition of `cJSON_AddItemReferenceToObject'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2070: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_AddNullToObject':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2080: multiple definition of `cJSON_AddNullToObject'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2080: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_CreateNull':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2386: multiple definition of `cJSON_CreateNull'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2386: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_AddTrueToObject':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2092: multiple definition of `cJSON_AddTrueToObject'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2092: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_CreateTrue':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2397: multiple definition of `cJSON_CreateTrue'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2397: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_AddFalseToObject':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2104: multiple definition of `cJSON_AddFalseToObject'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2104: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_CreateFalse':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2408: multiple definition of `cJSON_CreateFalse'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2408: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_AddBoolToObject':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2116: multiple definition of `cJSON_AddBoolToObject'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2116: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_CreateBool':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2419: multiple definition of `cJSON_CreateBool'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2419: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_AddNumberToObject':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2128: multiple definition of `cJSON_AddNumberToObject'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2128: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_CreateNumber':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2430: multiple definition of `cJSON_CreateNumber'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2430: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_AddStringToObject':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2140: multiple definition of `cJSON_AddStringToObject'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2140: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_CreateString':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2456: multiple definition of `cJSON_CreateString'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2456: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_AddRawToObject':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2152: multiple definition of `cJSON_AddRawToObject'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2152: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_CreateRaw':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2506: multiple definition of `cJSON_CreateRaw'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2506: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_AddObjectToObject':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2164: multiple definition of `cJSON_AddObjectToObject'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2164: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_CreateObject':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2534: multiple definition of `cJSON_CreateObject'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2534: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_AddArrayToObject':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2176: multiple definition of `cJSON_AddArrayToObject'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2176: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_CreateArray':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2523: multiple definition of `cJSON_CreateArray'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2523: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_DetachItemViaPointer':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2188: multiple definition of `cJSON_DetachItemViaPointer'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2188: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_DetachItemFromArray':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2224: multiple definition of `cJSON_DetachItemFromArray'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2224: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_DeleteItemFromArray':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2234: multiple definition of `cJSON_DeleteItemFromArray'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2234: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_DetachItemFromObject':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2239: multiple definition of `cJSON_DetachItemFromObject'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2239: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_DetachItemFromObjectCaseSensitive':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2246: multiple definition of `cJSON_DetachItemFromObjectCaseSensitive'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2246: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_DeleteItemFromObject':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2253: multiple definition of `cJSON_DeleteItemFromObject'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2253: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_DeleteItemFromObjectCaseSensitive':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2258: multiple definition of `cJSON_DeleteItemFromObjectCaseSensitive'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2258: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_InsertItemInArray':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2264: multiple definition of `cJSON_InsertItemInArray'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2264: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_ReplaceItemViaPointer':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2293: multiple definition of `cJSON_ReplaceItemViaPointer'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2293: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_ReplaceItemInArray':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2342: multiple definition of `cJSON_ReplaceItemInArray'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2342: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_ReplaceItemInObject':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2375: multiple definition of `cJSON_ReplaceItemInObject'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2375: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_ReplaceItemInObjectCaseSensitive':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2380: multiple definition of `cJSON_ReplaceItemInObjectCaseSensitive'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2380: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_CreateStringReference':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2473: multiple definition of `cJSON_CreateStringReference'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2473: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_CreateObjectReference':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2485: multiple definition of `cJSON_CreateObjectReference'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2485: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_CreateArrayReference':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2495: multiple definition of `cJSON_CreateArrayReference'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2495: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_CreateIntArray':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2546: multiple definition of `cJSON_CreateIntArray'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2546: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_CreateFloatArray':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2586: multiple definition of `cJSON_CreateFloatArray'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2586: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_CreateDoubleArray':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2626: multiple definition of `cJSON_CreateDoubleArray'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2626: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_CreateStringArray':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2666: multiple definition of `cJSON_CreateStringArray'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2666: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_Duplicate':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2707: multiple definition of `cJSON_Duplicate'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2707: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_Minify':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2839: multiple definition of `cJSON_Minify'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2839: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_IsInvalid':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2887: multiple definition of `cJSON_IsInvalid'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2887: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_IsFalse':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2897: multiple definition of `cJSON_IsFalse'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2897: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_IsTrue':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2907: multiple definition of `cJSON_IsTrue'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2907: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_IsBool':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2918: multiple definition of `cJSON_IsBool'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2918: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_IsNull':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2927: multiple definition of `cJSON_IsNull'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2927: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_IsArray':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2957: multiple definition of `cJSON_IsArray'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2957: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_IsObject':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2967: multiple definition of `cJSON_IsObject'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2967: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_IsRaw':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2977: multiple definition of `cJSON_IsRaw'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2977: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_Compare':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2987: multiple definition of `cJSON_Compare'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:2987: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o: in function `cJSON_malloc':
/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:3112: multiple definition of `cJSON_malloc'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cJSON.o:/home/jpn323/workspace/gh96-ut-core/src/cJSON.c:3112: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/ut_cunit.o:/home/jpn323/workspace/gh96-ut-core/src/cunit/ut_cunit.c:52: multiple definition of `group_list'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/ut_cunit.o:/home/jpn323/workspace/gh96-ut-core/src/cunit/ut_cunit.c:52: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/ut_cunit.o: in function `startup_system':
/home/jpn323/workspace/gh96-ut-core/src/cunit/ut_cunit.c:70: multiple definition of `startup_system'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/ut_cunit.o:/home/jpn323/workspace/gh96-ut-core/src/cunit/ut_cunit.c:70: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/ut_cunit.o: in function `UT_set_test_mode':
/home/jpn323/workspace/gh96-ut-core/src/cunit/ut_cunit.c:89: multiple definition of `UT_set_test_mode'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/ut_cunit.o:/home/jpn323/workspace/gh96-ut-core/src/cunit/ut_cunit.c:89: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/ut_cunit.o: in function `UT_Manage_Suite_Activation':
/home/jpn323/workspace/gh96-ut-core/src/cunit/ut_cunit.c:103: multiple definition of `UT_Manage_Suite_Activation'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/ut_cunit.o:/home/jpn323/workspace/gh96-ut-core/src/cunit/ut_cunit.c:103: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/ut_cunit.o: in function `UT_run_tests':
/home/jpn323/workspace/gh96-ut-core/src/cunit/ut_cunit.c:124: multiple definition of `UT_run_tests'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/ut_cunit.o:/home/jpn323/workspace/gh96-ut-core/src/cunit/ut_cunit.c:124: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/ut_cunit.o: in function `UT_toggle_suite_activation_based_on_groupID':
/home/jpn323/workspace/gh96-ut-core/src/cunit/ut_cunit.c:246: multiple definition of `UT_toggle_suite_activation_based_on_groupID'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/ut_cunit.o:/home/jpn323/workspace/gh96-ut-core/src/cunit/ut_cunit.c:246: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/ut_cunit.o: in function `UT_add_suite':
/home/jpn323/workspace/gh96-ut-core/src/cunit/ut_cunit.c:193: multiple definition of `UT_add_suite'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/ut_cunit.o:/home/jpn323/workspace/gh96-ut-core/src/cunit/ut_cunit.c:193: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/ut_cunit.o: in function `UT_add_suite_withGroupID':
/home/jpn323/workspace/gh96-ut-core/src/cunit/ut_cunit.c:198: multiple definition of `UT_add_suite_withGroupID'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/ut_cunit.o:/home/jpn323/workspace/gh96-ut-core/src/cunit/ut_cunit.c:198: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/ut_cunit.o: in function `UT_toggle_all_suites':
/home/jpn323/workspace/gh96-ut-core/src/cunit/ut_cunit.c:262: multiple definition of `UT_toggle_all_suites'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/ut_cunit.o:/home/jpn323/workspace/gh96-ut-core/src/cunit/ut_cunit.c:262: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/ut_cunit.o: in function `UT_add_test':
/home/jpn323/workspace/gh96-ut-core/src/cunit/ut_cunit.c:270: multiple definition of `UT_add_test'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/ut_cunit.o:/home/jpn323/workspace/gh96-ut-core/src/cunit/ut_cunit.c:270: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/ut_cunit.o: in function `UT_getTestSuiteTitle':
/home/jpn323/workspace/gh96-ut-core/src/cunit/ut_cunit.c:285: multiple definition of `UT_getTestSuiteTitle'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/ut_cunit.o:/home/jpn323/workspace/gh96-ut-core/src/cunit/ut_cunit.c:285: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/cunit_lgpl/ut_basic.o: in function `UT_basic_run_tests':
/home/jpn323/workspace/gh96-ut-core/src/cunit/cunit_lgpl/ut_basic.c:83: multiple definition of `UT_basic_run_tests'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/cunit_lgpl/ut_basic.o:/home/jpn323/workspace/gh96-ut-core/src/cunit/cunit_lgpl/ut_basic.c:83: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/cunit_lgpl/ut_basic.o: in function `UT_basic_run_suite':
/home/jpn323/workspace/gh96-ut-core/src/cunit/cunit_lgpl/ut_basic.c:99: multiple definition of `UT_basic_run_suite'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/cunit_lgpl/ut_basic.o:/home/jpn323/workspace/gh96-ut-core/src/cunit/cunit_lgpl/ut_basic.c:99: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/cunit_lgpl/ut_basic.o: in function `UT_basic_run_test':
/home/jpn323/workspace/gh96-ut-core/src/cunit/cunit_lgpl/ut_basic.c:112: multiple definition of `UT_basic_run_test'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/cunit_lgpl/ut_basic.o:/home/jpn323/workspace/gh96-ut-core/src/cunit/cunit_lgpl/ut_basic.c:112: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/cunit_lgpl/ut_basic.o: in function `UT_basic_set_mode':
/home/jpn323/workspace/gh96-ut-core/src/cunit/cunit_lgpl/ut_basic.c:127: multiple definition of `UT_basic_set_mode'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/cunit_lgpl/ut_basic.o:/home/jpn323/workspace/gh96-ut-core/src/cunit/cunit_lgpl/ut_basic.c:127: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/cunit_lgpl/ut_basic.o: in function `UT_basic_get_mode':
/home/jpn323/workspace/gh96-ut-core/src/cunit/cunit_lgpl/ut_basic.c:133: multiple definition of `UT_basic_get_mode'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/cunit_lgpl/ut_basic.o:/home/jpn323/workspace/gh96-ut-core/src/cunit/cunit_lgpl/ut_basic.c:133: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/cunit_lgpl/ut_basic.o: in function `UT_basic_show_failures':
/home/jpn323/workspace/gh96-ut-core/src/cunit/cunit_lgpl/ut_basic.c:139: multiple definition of `UT_basic_show_failures'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/cunit_lgpl/ut_basic.o:/home/jpn323/workspace/gh96-ut-core/src/cunit/cunit_lgpl/ut_basic.c:139: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/cunit_lgpl/ut_automated.o: in function `UT_set_results_output_filename':
/home/jpn323/workspace/gh96-ut-core/src/cunit/cunit_lgpl/ut_automated.c:108: multiple definition of `UT_set_results_output_filename'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/cunit_lgpl/ut_automated.o:/home/jpn323/workspace/gh96-ut-core/src/cunit/cunit_lgpl/ut_automated.c:108: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/cunit_lgpl/ut_automated.o: in function `UT_automated_run_tests':
/home/jpn323/workspace/gh96-ut-core/src/cunit/cunit_lgpl/ut_automated.c:141: multiple definition of `UT_automated_run_tests'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/cunit_lgpl/ut_automated.o:/home/jpn323/workspace/gh96-ut-core/src/cunit/cunit_lgpl/ut_automated.c:141: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/cunit_lgpl/ut_automated.o: in function `UT_list_tests_to_file':
/home/jpn323/workspace/gh96-ut-core/src/cunit/cunit_lgpl/ut_automated.c:175: multiple definition of `UT_list_tests_to_file'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/cunit_lgpl/ut_automated.o:/home/jpn323/workspace/gh96-ut-core/src/cunit/cunit_lgpl/ut_automated.c:175: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/cunit_lgpl/ut_automated.o: in function `UT_automated_list_all_tests':
/home/jpn323/workspace/gh96-ut-core/src/cunit/cunit_lgpl/ut_automated.c:656: multiple definition of `UT_automated_list_all_tests'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/cunit_lgpl/ut_automated.o:/home/jpn323/workspace/gh96-ut-core/src/cunit/cunit_lgpl/ut_automated.c:656: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/cunit_lgpl/ut_automated.o: in function `UT_automated_package_name_get':
/home/jpn323/workspace/gh96-ut-core/src/cunit/cunit_lgpl/ut_automated.c:795: multiple definition of `UT_automated_package_name_get'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/cunit_lgpl/ut_automated.o:/home/jpn323/workspace/gh96-ut-core/src/cunit/cunit_lgpl/ut_automated.c:795: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/cunit_lgpl/ut_automated.o: in function `UT_automated_enable_junit_xml':
/home/jpn323/workspace/gh96-ut-core/src/cunit/cunit_lgpl/ut_automated.c:772: multiple definition of `UT_automated_enable_junit_xml'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/cunit_lgpl/ut_automated.o:/home/jpn323/workspace/gh96-ut-core/src/cunit/cunit_lgpl/ut_automated.c:772: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/cunit_lgpl/ut_automated.o: in function `UT_automated_package_name_set':
/home/jpn323/workspace/gh96-ut-core/src/cunit/cunit_lgpl/ut_automated.c:781: multiple definition of `UT_automated_package_name_set'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/cunit_lgpl/ut_automated.o:/home/jpn323/workspace/gh96-ut-core/src/cunit/cunit_lgpl/ut_automated.c:781: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/cunit_lgpl/ut_console.o: in function `UT_console_run_tests':
/home/jpn323/workspace/gh96-ut-core/src/cunit/cunit_lgpl/ut_console.c:120: multiple definition of `UT_console_run_tests'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/cunit/cunit_lgpl/ut_console.o:/home/jpn323/workspace/gh96-ut-core/src/cunit/cunit_lgpl/ut_console.c:120: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/ut_kvp_profile.o: in function `ut_kvp_profile_open':
/home/jpn323/workspace/gh96-ut-core/src/ut_kvp_profile.c:26: multiple definition of `ut_kvp_profile_open'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/ut_kvp_profile.o:/home/jpn323/workspace/gh96-ut-core/src/ut_kvp_profile.c:26: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/ut_kvp_profile.o: in function `ut_kvp_profile_close':
/home/jpn323/workspace/gh96-ut-core/src/ut_kvp_profile.c:42: multiple definition of `ut_kvp_profile_close'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/ut_kvp_profile.o:/home/jpn323/workspace/gh96-ut-core/src/ut_kvp_profile.c:42: first defined here
/usr/bin/ld: /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/ut_kvp_profile.o: in function `ut_kvp_profile_getInstance':
/home/jpn323/workspace/gh96-ut-core/src/ut_kvp_profile.c:51: multiple definition of `ut_kvp_profile_getInstance'; /home/jpn323/workspace/gh96-ut-core/tests/build/linux/obj/src/ut_kvp_profile.o:/home/jpn323/workspace/gh96-ut-core/src/ut_kvp_profile.c:51: first defined here
collect2: error: ld returned 1 exit status
make[2]: *** [Makefile:123: test] Error 1
make[2]: Leaving directory '/home/jpn323/workspace/gh96-ut-core'
make[1]: *** [Makefile:118: framework] Error 2
make[1]: Leaving directory '/home/jpn323/workspace/gh96-ut-core'
make: *** [Makefile:70: framework] Error 2
make: Leaving directory '/home/jpn323/workspace/gh96-ut-core/tests'
jpn323@janus ~/workspace/gh96-ut-core (feature/gh96-separate-obj-dir-ut-core)$ 
jpn323@janus ~/workspace/gh96-ut-core (feature/gh96-separate-obj-dir-ut-core)$ 

```

Reply:
ok but sort can't possibly fix that.

```bash
 cat hello.txt 
a
b
c
d
e
f
f
e
d
c
b
a
```

...

```bash
 cat hello.txt | sort
a
a
b
b
c
c
d
d
e
e
f
f
```

Reply:
Makefile behaves differently from the sort command used in a Linux shell, although both are used to order items and remove duplicates.

1. Makefile sort Function:
In a Makefile, $(sort ...) is a built-in function that sorts words in lexical order and automatically removes duplicates. It only works on a space-separated list of words, and the operation happens entirely within Makefile syntax (not by invoking an external command).

Syntax: $(sort list)
Behavior: Sorts the items in list and removes duplicates.
Output: A space-separated list of unique, sorted items.
Example:

Ex makefile
```makefile
OBJS := file3.o file1.o file2.o file1.o
OBJS := $(sort $(OBJS))
# OBJS will now be "file1.o file2.o file3.o"

```